sel4test-sim/builds.yml

```diff
@@ -65,7 +65,7 @@ build-filter:
 #      - arch: [x86, riscv]
 #        compiler: [gcc]
 #        mode: [32]
-#        simulation_binary:
+#        has_simulation:
```

Comment:
To make it clearer, let's bring the example in sync with the use above.

```suggestion
#        has_simulation: true
```

Reply:
done